Form/LegacyModifySettingsForm.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace Craue\ConfigBundle\Form;
+
+/**
+ * for Symfony < 2.8
+ *
+ * @author Christian Raue <christian.raue@gmail.com>
+ * @copyright 2011-2015 Christian Raue
+ * @license http://opensource.org/licenses/mit-license.php MIT License
+ */
+class LegacyModifySettingsForm extends AbstractModifySettingsForm {
```

Comment:
you could get rid of the abstract class by making the legacy class extend the new class rather than an parent containing all the code